edgeless_orc/src/proxy_redis.rs

```diff
@@ -56,11 +56,11 @@ impl ProxyRedis {
         );
 
         // create the connection with the Redis server
-        let mut connection = redis::Client::open(redis_url)?.get_connection()?;
+        let mut connection: redis::Connection = redis::Client::open(redis_url)?.get_connection()?;
 
         if flushdb {
             // flush the in-memory database upon construction
-            redis::cmd("FLUSHDB").query(&mut connection)?;
+            let _: () = redis::cmd("FLUSHDB").query(&mut connection)?;
```

Comment:
I see this fixes the warning you got on newer compiler, but I think it would be more correct to specify the type of the query() method, like this:

```
let _ = redis::cmd("FLUSHDB").query::<String>(&mut connection)?;
```